projects/vdk-plugins/vdk-structlog/src/vdk/plugin/structlog/constants.py

```diff
@@ -4,6 +4,7 @@
 
 STRUCTLOG_LOGGING_METADATA_KEY = "logging_metadata"
 STRUCTLOG_LOGGING_FORMAT_KEY = "logging_format"
+STRUCTLOG_LOGGING_CUSTOM_FORMAT_KEY = "logging_custom_format"
```

Comment:
This is applicable only for Console format. 

So really it should be STRUCTLOG_CONSOLE_LOG_PATTERN 

The name is inspired by the similar variable in spring logging - https://docs.spring.io/spring-boot/docs/2.1.13.RELEASE/reference/html/boot-features-logging.html